.eslintrc.js

```diff
@@ -23,28 +24,20 @@ module.exports = {
   },
 
   settings: {
-    'import/core-modules': ['require', 'backburner', 'router', 'ember/version', 'node-module'],
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts'],
-    },
```

Comment:
This is handled by [`plugin:import/typescript`](https://github.com/benmosher/eslint-plugin-import/blob/c51b6a97de1e0edad77a247d5da4c36db7c8d039/config/typescript.js#L12-L14)